pkg/statistics/handle/autoanalyze/autoanalyze.go

```diff
@@ -494,7 +494,7 @@ func tryAutoAnalyzeTable(
 	// 1. If the stats are not loaded, we don't need to analyze it.
 	// 2. If the table is too small, we don't want to waste time to analyze it.
 	//    Leave the opportunity to other bigger tables.
-	if statsTbl.Pseudo || statsTbl.RealtimeCount < AutoAnalyzeMinCnt {
+	if statsTbl == nil || statsTbl.RealtimeCount < AutoAnalyzeMinCnt || statsTbl.Pseudo {
```

Comment:
Please update the comments to explain when the statsTbl is not nil but it is pseudo.

Reply:
I think it is forgotten to delete.

Reply:
It should just change the comment.